src/AttributesAndMethods.jl

```diff
@@ -308,61 +526,71 @@ end
 export cox_ring
 
 
+"""
+    Base.:*( v::NormalToricVariety, w::NormalToricVariety )
+
+Compute the direct product of the normal toric varieties `v` and `w`.
+
+# Examples
+```julia-repl
+julia> projective_space( 2 ) * projective_space( 2 )
+NormalToricVariety(GAP: <A projective toric variety of dimension 4 which is a product of 2 toric varieties>, Polymake.BigObjectAllocated(Ptr{Nothing} @0x00005645a1a00930))
+```
+"""
 function Base.:*( v::NormalToricVariety, w::NormalToricVariety )
     gap_NormalToricVariety = v.GapNTV * w.GapNTV
     return NormalToricVariety( gap_NormalToricVariety )
 end
 export *
 
-#= 
-
-#! @Description
-#!  Computes the rational function corresponding to the character grid element <A>elem</A> or to the list of 
-#!  integers <A>elem</A>. This computation needs to know the coordinate ring of the torus of the variety <A>vari</A>. By
-#!  default this ring is introduced with variables <A>x1</A> to <A>xn</A> where <A>n</A> is the dimension of the variety. If
-#!  different variables should be used, then <A>CoordinateRingOfTorus</A> has to be set accordingly before calling this method.
-#! @Returns a homalg element
-#! @Arguments elem, vari
-DeclareOperation( "CharacterToRationalFunction",
-                  [ IsList, IsNormalToricVariety ] );
-
-#! @Description
-#!  Returns a list of the currently defined Divisors of the toric variety.
-#! @Returns a list
-#! @Arguments vari
-DeclareOperation( "WeilDivisorsOfVariety",
-                  [ IsNormalToricVariety ] );
-
-#! @Description
-#!  
-#! @Arguments vari
-DeclareOperation( "Factors",
-                  [ IsNormalToricVariety ] );
-
-#! @Description
-#!  
-#! @Arguments vari, p
-DeclareOperation( "BlowUpOnIthMinimalTorusOrbit",
-                  [ IsNormalToricVariety, IsInt ] );
-
-#! @Description
-#!
-DeclareGlobalFunction( "ZariskiCotangentSheafViaEulerSequence" );
-
-#! @Description
-#!
-DeclareGlobalFunction( "ZariskiCotangentSheafViaPoincareResidueMap" );
-
-#! @Description
-#!  
-#! @Arguments vari, p
-DeclareOperation( "ithBettiNumber",
-                  [ IsNormalToricVariety, IsInt ] );
-
-#! @Description
-#!  
-#! @Arguments vari, p
-DeclareOperation( "NrOfqRationalPoints",
-                  [ IsNormalToricVariety, IsInt ] );
-
-=#
+
+struct character_to_rational_function
```

Comment:
Struct names really should follow CamelCase.....

Reply:
I'll adjust that.